src/django/api/models.py

```diff
@@ -873,8 +873,9 @@ class FacilityClaim(models.Model):
         null=True,
         default=None,
         on_delete=models.PROTECT,
-        verbose_name='parent company',
-        help_text='The parent company of this facility claim.')
+        verbose_name='parent company / supplier group',
+        help_text='The parent company / supplier group of this'
+        'facility claim.')
```

Comment:
There is a missing space between `this` and `facility`. Unfortunately this means that, when it is corrected, you will need to recreate the migration. The easiest way to do this is to:
- Unapply the migration with `./scripts/manage migrate api 0042`
- Delete 0043_auto_20200303_1837.py
- Fix the help text
- Rerun `./scripts/manage makemigrations`

We like to rename the automatically generated migration files so that you can see at a glance what they do. After recreating the migration, consider renaming it to `0043_facility_claim_parent_company_verbose_name_change.py`

Reply:
fixed in 5d0c66b0f2ef1941402f825469aaf174ca354f89